crates/matrix-sdk/src/room/mod.rs

```diff
@@ -315,6 +316,24 @@ impl Room {
         self.client.subscribe_to_room_updates(self.room_id())
     }
 
+    /// Subscribe to typing notifications for this room.
+    ///
+    /// The returned receiver will receive a new vector of user IDs for each
+    /// sync response that contains 'm.typing' event.
+    pub fn subscribe_to_typing_notifications(
+        &self,
+    ) -> (EventHandlerDropGuard, broadcast::Receiver<Vec<OwnedUserId>>) {
+        let (sender, receiver) = broadcast::channel(16);
```

Comment:
Why 16? Why not an unbounded channel?

Reply:
I don't think we need an unbounded channel for such usage, we won't have more than subscribers. What do you think?

Reply:
I'm fine with 16. Can you add a comment please to justify this number?